lib/stream_rails/sync_policies.rb

```diff
@@ -5,7 +5,20 @@ module SyncPolicy
     module SyncCreate
 
       def self.included(base)
-        base.after_commit :add_to_feed, on: :create
+        if base.respond_to? :after_commit
+          base.after_commit :add_to_feed, on: :create
+        else
+          base.instance_eval do
+            define_method(:add_to_feed) {
```

Comment:
not sure I get it, can you add a code snippet on how you hook save/delete callbacks with Sequel?